_data/matrix/Bitcoin_Core_Wallet.yaml

```diff
@@ -0,0 +1,36 @@
+---
+name: "Bitcoin Core Wallet"
+category: "Software Wallet (L1)"
+keyfeatures: "Open source project which maintains and releases Bitcoin client software called Bitcoin Core."
+homepage: "https://bitcoincore.org"
+
+feature:
+  device: "Desktop"
+  os: "Linux, OS, Windows"
+  web: "No"
+  hw_interface: "Yes"
+  default_receive: "P2WPKH"
+  bech32: "Full Support (send & receive)"
+  bech32m: "Full Support (send & receive)"
+  segwit_change: "Yes"
+  full_rbf_bump_fee: "Yes"
+  full_rbf_cancel_txn: "Yes"
+  full_rbf_notification: "No"
+  cpfp: "Full Support (send & receive)"
+  descriptors: "Yes"
+  pbst_version: "V0"
+  pbst_export: "Yes"
+  pbst_update_finalize: "Yes"
+  musig2: "No Support"
+  coinjoin: "No Support"
+  payjoin_BIP78: "No Support"
+  payjoin_alternate: "No Support"
+  payjoin_alternate_name: ""
+  payment_code: "No Support"
+  silent_payments_BIP352: "No Support"
+  silent_payments_dns: "No Support"
+  lightning_11: "No Support"
+  lightning_12: "No Support"
+  lightning_12_dns: "No Support"
+  lightning_lnurl: "No Support"
+  lightning_lnurl_hrla: "No Support"
```

Comment:
Shouldn’t this all be "Not Applicable" for an L1 wallet?

Reply:
Agreed.  That was a silly mistake on my part. I will update.